test/models/setting_test.rb

```diff
@@ -600,4 +600,14 @@ def sticky_setting
     refute_empty sorted_list
     assert sorted_list.keys.exclude?(sticky_setting)
   end
+
+  test 'full_name_with_default should include default value' do
+    setting = Setting.create(:name => 'foo', :value => 'baz', :default => 'boo', :description => 'test foo', :full_name => 'Foo Name')
```

Comment:
Can you try `Factory.build` to avoid creating an object in db (slower)?

Reply:
Agreed. Done.